ui/src/message_popup/renderers/audio_response_summary.jsx

```diff
@@ -42,10 +44,12 @@ export default React.createClass({
           </div>
           <p style={styles.summaryTitle}>Summary</p>
           <Divider />
-          {audioResponses.map((obj, i) =>
-            <div key={i} style ={_.merge(styles.instructions)}>
-              <ReadMore fulltext={obj.questionText} />
-              <audio controls={true} src={obj.audioUrl} />
+          {audioResponses.map((audioResponse) =>
+            <div key={audioResponse.questionId} style ={_.merge(styles.instructions, styles.summaryQuestion)}>
```

Comment:
You can also use destructuring to merge to objects like `style={{...styles.instructions, ...styles.summaryQuestion}}` in place of `_.merge`.  That's more in the style of JSX and ES6 and I hadn't shifted over in older code.